src/PrestaShopBundle/Form/Admin/Sell/Product/FooterType.php

```diff
@@ -148,6 +149,18 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 ],
             ])
             // These two inputs are displayed separately
+            ->add('preview', IconButtonType::class, [
+                'label' => $this->trans('Preview', 'Admin.Actions'),
+                'icon' => 'remove_red_eye',
```

Comment:
Oh great, you left the icons! Can you please keep them for now? In the mockups all the icons have been removed but I'd like to discuss this with @TristanLDD and show him both possibilities in live to compare, I personally prefer with the icons 😅

But this will be decided during the UX review of the page.

Reply:
I've not seen that been removed on mockup ... lets wait for @TristanLDD

Reply:
I vote for comeback of icons!

Reply:
We have planned to do a full review of the UX once the page is iso-functional. SO I'm not sure we need to wait for @TristanLDD to merge this PR. But if he answers before the merge we can integrate his recommendations of course ^^

Reply:
I commented all icon to validate if the look and feel is better... so we can validate with removing these or rollback uncommenting